Extensions/DraggablePhysics.json

```diff
@@ -0,0 +1,554 @@
+{
+  "author": "",
+  "category": "",
+  "description": "Enables users to click and drag on objects to move them.  The object will retain velocity when the click is released, which can allow users to fling objects across the screen.\n\nNote:\n- The standard \"draggable\" behavior will not work on objects that have the physics behavior. This extension should be used instead.\n- This extension will only work on objects that have the physics behavior.",
+  "extensionNamespace": "",
+  "fullName": "Draggable (for physics objects)",
+  "helpPath": "",
+  "iconUrl": "",
```

Comment:
You didn't choose an icon for this new extension?
I see the Laser-pointer icon, feel free to choose one.

Reply:
Good catch! I picked one and pushed an update.  

Thanks for the review, Bouh!